catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/TransformersCommand.java

```diff
@@ -38,9 +41,9 @@
 
   // output strings
 
-  static final String ACTIVE_TRANSFORMERS_HEADER = "Active Transformers: ";
+  protected static final String METACARD = "Metacard";
 
```

Comment:
Did you need to change this to protected, or is package private okay?

Reply:
package private would have worked I just changed it because codacy was complaining that everything should have a specified scope.